CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Updated to Angular 9
 
 ### Added
+- Link between text and images
```

Comment:
This message is a little bit misleading: it seems that you have implemented the Image Text Linking feature. 
Change into something like "Text/images connection in page change"